### ISSUE TITLE
date and Day is not maching while Change in time Zone

### ISSUE THREAD

**Issue by bhargavagonugunta** [calcom/cal.com#11648]
Found a bug? Please fill out the sections below. 👍

### Issue Summary
issue is in booking page when we change the zone in the admin page  date and day is not matching correctly

A summary of the issue. This needs to be a clear detailed-rich summary.

### Steps to Reproduce

1. Went to settings 
2. go to General
3. make a change in Time Zone 
4. save 
5. go to booking link

Loom Video for more Info
https://www.loom.com/share/8034d4acbf7e4688922d918f4dbf807a?sid=df9e8cbd-0ca9-420b-acaa-c01d928424f8

### Actual Results
date and day is not matching correctly in booking page

### Expected Results

date and day is must mach with Real Time

**Comment by PeerRich** [comment]
the two timezones are unrelated. one is for your internal dashboard and the booking page depends on the person opening it.

**Comment by PeerRich** [comment]
tldr: not a bug
